apps/cli/src/deploy.ts

```diff
@@ -441,8 +441,9 @@ const deploy = async (
                     return true;
                 },
             });
+            const payee = await factory.read.payee();
             const { request } = await token.simulate.approve(
-                [factory.address, newAllowance],
+                [payee, newAllowance],
```

Comment:
The tokens might be going to the payee now, but they are still managed by the factory, right?
As a result, this code section shouldn't need to change at all.

From [EIP-20](https://eips.ethereum.org/EIPS/eip-20):

> The `transferFrom` method is used for a withdraw workflow, allowing contracts to transfer tokens on your behalf. [...] The function SHOULD `throw` unless the `_from` account has deliberately authorized the sender of the message via some mechanism.

>

Reply:
Exactly, the recipient is not relevant in this case.
You can see how this specification is implemented by OpenZeppelin [here](https://github.com/OpenZeppelin/openzeppelin-contracts/blob/a5ed318634016a25be4000ee07044a31f363e60c/contracts/token/ERC20/ERC20.sol#L163-L168).